docs/_includes/header.html

```diff
@@ -9,12 +9,16 @@
 
         {% if site-navigation %}
             <nav aria-label="Global" class="sm:d-none">
-                <ul class="s-navigation ml8 fw-nowrap sm:d-none">
```

Comment:
The removal of `sm:d-none` here has nothing to do with the navigation component changes but the class is redundant here since the parent includes it too so I figured I might as well chuck it in the bin now.